File: Tests/installation_tests/cocoapods/without_frameworks_objc/CocoapodsTest/ViewController.m
//
//  ViewController.m
//  CocoapodsTest
//
//  Created by Jack Flintermann on 8/4/15.
//  Copyright (c) 2015 jflinter. All rights reserved.
//

#import "ViewController.h"
@import Stripe;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [StripeAPI setDefaultPublishableKey:@"test"];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Tests/installation_tests/swift_package_manager/with_swift/SPMTest/ViewController.swift
//
//  ViewController.swift
//  SPMTest
//
//  Created by Mel Ludowise on 8/3/21.
//  Copyright © 2021 Stripe. All rights reserved.
//

import Stripe
import StripeApplePay
import StripeCardScan
import StripeFinancialConnections
import StripeIdentity
import StripePaymentSheet
import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        StripeAPI.defaultPublishableKey = "foo"

        if #available(iOS 14.3, *) {
            let _ = IdentityVerificationSheet(verificationSessionClientSecret: "test")
        }

        if #available(iOS 12.0, *) {
            let _ = FinancialConnectionsSheet(
                financialConnectionsSessionClientSecret: "",
                returnURL: nil
            )
        }

        // Initialize a card field to make sure we can load image resources
        let cardField = STPPaymentCardTextField()
        cardField.frame = CGRect(x: 0, y: 0, width: 300, height: 100)
        self.view.addSubview(cardField)
        let _ = CardImageVerificationSheet(
            cardImageVerificationIntentId: "foo",
            cardImageVerificationIntentSecret: "foo"
        )

        let _ = PaymentSheet(
            setupIntentClientSecret: "",
            configuration: PaymentSheet.Configuration()
        )
        // Do any additional setup after loading the view.

    }
}
